lib/rspec/que/queue_up.rb

```diff
@@ -1,89 +1,155 @@
 require 'rspec/mocks/argument_list_matcher'
+require 'time'
 
 module RSpec
   module Que
     module Matchers
       class QueueUp
         def initialize(job_class = nil)
+          @matchers = [QueuedSomething.new]
+          @matchers << QueuedClass.new(job_class) if job_class
           @job_class = job_class
+
+          @stages = nil
```

Comment:
Ok:
```
@matchers = [QueuedSomething.new]
@matchers << QueuedClass.new(job_class) if job_class

@job_class = job_class
@stages = []
```
This would look good. Also, address the `before_jobs` comment to make sure we don't save other instance variables.